installer/rpm/algorand/algorand.spec

```diff
@@ -5,6 +5,7 @@ Summary:       Algorand node software
 URL:           https://www.algorand.com
 License:       AGPL-3+
 Requires:      yum-cron
+
```

Comment:
we probably need another spec that requires `dnf-cron` right?

Reply:
Yes, it is` installer/rpm/algorand/algorand-centos8-stream.spec`, the file before this one.